src/Tasks/GenerateLauncher.cs

```diff
@@ -39,7 +39,7 @@ public sealed class GenerateLauncher : TaskExtension
 
         public override bool Execute()
         {
-            if (LauncherPath == null)
+            if (LauncherPath == null && NativeMethodsShared.IsWindows)
```

Comment:
Is this meant as a "don't run on windows"? Attribute to disable it would be much better here or making it more effectively crash.

Reply:
Somehow I skipped over this one. It's part of clickonce, so I'll mark the class as windows only and log "NotSupported" if Execute is called and we're not on windows.